Leanplum/Leanplum.m

```diff
@@ -475,6 +499,37 @@ + (void)startWithUserId:(NSString *)userId
                     }
                 }];
     
+    // Resume.
+    [[NSNotificationCenter defaultCenter]
+        addObserverForName:UIApplicationWillEnterForegroundNotification
+                    object:nil
+                     queue:nil
+                usingBlock:^(NSNotification *notification) {
+                    if ([[UIApplication sharedApplication]
+                            respondsToSelector:@selector(currentUserNotificationSettings)]) {
+                        [[LPActionManager sharedManager] sendUserNotificationSettingsIfChanged:
+                                                             [[UIApplication sharedApplication]
+                                                                 currentUserNotificationSettings]];
+                    }
+                    [Leanplum resume];
+                    if (startedInBackground && !inForeground) {
+                        inForeground = YES;
+                        /*[self maybePerformActions:@[@"start", @"resume"]
```

Comment:
please can we put descriptive todos here? I think it's fine to comment this out, but having a clear todo could help when we re-visit